doc/data/messages/c/cyclic-import/bad.py

```diff
@@ -0,0 +1,9 @@
+cat > a.py <<EOF
```

Comment:
This is not valid python, it can't work. I think cyclic-import is not the easiest message to document as it implicate multiple files.

Reply:
I got the documentation from pylint-errors, which is where I was supposed to get it from for: https://github.com/PyCQA/pylint/issues/5953

Reply:
We added some more checks and normalization around it. Most of the message are already implemented so the remaining one can be a little hard to do (like cyclic-import or duplicate code). It's ok if you don't want /can't handle the hard one :)